### PR TITLE
fix: fix output formats

### LINKED ISSUE #1659: Fix formatting of service error message
```
2019/12/14 03:01:06 service[kubelet](Waiting): Error running Containerd(kubelet), going to restart forever: %!w(*errors.errorString=&{task "kubelet" failed: exit code 255})
```

### PR DESCRIPTION
**Pull request by andrewrynhard**
This fixes random log issues.

Fixes #1659.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -46,6 +49,7 @@ require (
 	github.com/mdlayher/genetlink v0.0.0-20190313224034-60417448a851
 	github.com/mdlayher/netlink v0.0.0-20191009155606-de872b0d824b
 	github.com/mdlayher/raw v0.0.0-20190606144222-a54781e5f38f // indirect
+	github.com/mitchellh/go-wordwrap v1.0.0 // indirect
```
Comment (smira) [inline go.mod]: where do the new deps come from?

Comment (andrewrynhard) [reply go.mod]: No clue... I was trying to figure that out. Modules are weird sometimes.

**Comment by bradbeam** [comment]
Want to grab https://github.com/talos-systems/talos/issues/1659?

File: go.mod (hunk 1 above)
Comment (andrewrynhard) [reply go.mod]: I forced a revert.

**Comment by andrewrynhard** [comment]
> Want to grab #1659?

Ahh yes. Thanks.

File: internal/app/networkd/pkg/networkd/networkd.go [hunk 2]
```diff
@@ -172,17 +172,17 @@ func (n *Networkd) Configure() (err error) {
 
 			// Ensure link exists
 			if err = netif.Create(); err != nil {
-				log.Println("error creating nic", err)
+				log.Printf("error creating nic: %v\n", err)
```
Comment (bradbeam) [inline internal/app/networkd/pkg/networkd/networkd.go]: nit, i don't think we want `\n` here

File: internal/app/machined/pkg/system/runner/restart/restart.go [hunk 3]
```diff
@@ -136,12 +136,12 @@ func (r *restarter) Run(eventSink events.Recorder) error {
 				return nil
 			}
 
-			eventSink(events.StateWaiting, "Error running %s, going to restart until it succeeds: %w", r.wrappedRunner, err)
+			eventSink(events.StateWaiting, "Error running %s, going to restart until it succeeds: %v", r.wrappedRunner, err)
```
Comment (smira) [inline internal/app/machined/pkg/system/runner/restart/restart.go]: :+1:

File: internal/app/networkd/pkg/networkd/networkd.go (hunk 2 above)
Comment (andrewrynhard) [reply internal/app/networkd/pkg/networkd/networkd.go]: We don't?

Comment (andrewrynhard) [reply internal/app/networkd/pkg/networkd/networkd.go]: https://play.golang.org/p/R5PHhPKwWI8

Comment (andrewrynhard) [reply internal/app/networkd/pkg/networkd/networkd.go]: Ahh, I guess that is not true with `log`: https://play.golang.org/p/k4dZ7GA3wnT

Comment (andrewrynhard) [reply internal/app/networkd/pkg/networkd/networkd.go]: Fixed.

**Pull request merged**